makefile

```diff
@@ -49,7 +53,10 @@ dotos: $(filenameso)
 #=======#
 
 clean:
-	rm libft.a test $(filenameso)
+	rm -f $(NAME)
+	rm -f $(filenameso)
```

Comment:
Could do it in one line.
Or maybe you'll want to comment out only one of the instructions someday, and it'll be a lot faster to do so with 2 separate instructions !